staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go

```diff
@@ -79,8 +79,8 @@ var (
 		# Check to see if I can create pods in any namespace
 		kubectl auth can-i create pods --all-namespaces
 
-		# Check to see if I can list deployments in my current namespace
-		kubectl auth can-i list deployments.apps
+		# Check to see if service account "foo" can list deployments in the current namespace
+		kubectl auth can-i list deployments.apps --as==system:serviceaccount:default:foo
```

Comment:
a few suggestions:
* add the impersonation example alongside the existing one rather than changing it
* note that it uses impersonation, so you must be authorized to impersonate that service account
* update the syntax to be `--as=...` (no double-equal)

Reply:
@liggitt thank you for your feedback. I updated the PR. Is it ok now?